musicrs/util/slack_api.py

```diff
@@ -69,3 +76,62 @@ def retrieve_slack_messages(channel: str, start_date: str, end_date: str):
                 retrieved_messages.append(retrieved_message)
 
     return retrieved_messages
+
+
+def post_slack_message(user_id, video_id):
+    """
+    Sends video url to the user.
+    param user_id: user's member id in slack workspace
+    type user_id: string
+    param video_id: youtube video id
+    type video_id: string
+    """
+    try:
+        dm_response = slackBotClient.conversations_open(users=user_id)
+        dm_channel = dm_response["channel"]["id"]
+        video_url = generate_url(video_id=video_id)
+
+        response = slackBotClient.chat_postMessage(channel=dm_channel, text=video_url)
```

Comment:
Change message fomat to: You might like this song $url which is $% similar to your interest.